gallery/src/styles/Nivo/data.tsx

```diff
@@ -0,0 +1,335 @@
+/*
+   * Copyright (c) 2019-present Sonatype, Inc.
+   * This program and the accompanying materials are made available under
+   * the terms of the Eclipse Public License 2.0 which accompanies this
+   * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+   */
+
+export const rawBarData: [string, number][] = [
+  ['foxes', 10],
+  ['wolves', 34],
+  ['dogs', 1],
+  ['cats', 5],
+  ['penguins', 43],
+  ['chewbaccas', 89],
+  ['weasels', 64],
+  ['raccons', 12],
+  ['pandas', 98],
+  ['deers', 27],
+  ['bears', 32]
+];
+
+export const pieData = [
+  {
+    'id': 'donuts',
+    'label': 'donuts',
+    'value': 357
+  },
+  {
+    'id': 'ice cream',
+    'label': 'ice cream',
+    'value': 411
+  },
+  {
+    'id': 'pizza',
+    'label': 'pizza',
+    'value': 528
+  },
+  {
+    'id': 'fries',
+    'label': 'fries',
+    'value': 232
+  },
+  {
+    'id': 'oreos',
+    'label': 'oreos',
+    'value': 222
+  },
+  {
+    'id': 'quesadilla',
+    'label': 'quesadilla',
+    'value': 411
+  },
+  {
+    'id': 'nachos',
+    'label': 'nachos',
+    'value': 328
+  },
+  {
+    'id': 'salad',
+    'label': 'salad',
+    'value': 86
+  }
+];
+
+export const lineData = [
+  {
+    'id': 'japan',
+    'data': [
+      {
+        'x': 'plane',
+        'y': 105
+      },
+      {
+        'x': 'helicopter',
+        'y': 40
+      },
+      {
+        'x': 'boat',
+        'y': 60
+      },
+      {
+        'x': 'train',
+        'y': 244
+      },
+      {
+        'x': 'subway',
+        'y': 132
+      },
+      {
+        'x': 'bus',
+        'y': 119
+      },
+      {
+        'x': 'car',
+        'y': 271
+      },
+      {
+        'x': 'moto',
+        'y': 237
+      },
+      {
+        'x': 'bicycle',
+        'y': 207
+      },
+      {
+        'x': 'horse',
+        'y': 278
+      },
+      {
+        'x': 'skateboard',
+        'y': 108
+      },
+      {
+        'x': 'others',
+        'y': 95
+      }
+    ]
+  },
+  {
+    'id': 'france',
+    'color': 'hsl(100, 70%, 50%)',
```

Comment:
What's this do?

Reply:
> What's this do?

The data had set colors for the values, but we have our colors. I forgot to remove it: 1cc03a3adaa451ff83e6ff217ed990f01c3106ed